src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/preview.html.twig

```diff
@@ -139,7 +139,7 @@
           <tbody>
           {% for productDetail in orderPreview.productDetails %}
             <tr class="{% if loop.index > productsPreviewLimit %}js-product-preview-more d-none{% endif %}">
-              <td class="p-1">{{ productDetail.name }}</td>
+              <td class="p-1"><a href="{{ path('admin_product_form', {'id': productDetail.id}) }}" target="_blank">{{ productDetail.name }}</a></td>
```

Comment:
The id is not mandatory, it will break if no one is passed

Reply:
Done